plugins/worker_api/include/golos/plugins/worker_api/worker_api_objects.hpp

```diff
@@ -75,7 +75,7 @@ namespace golos { namespace plugins { namespace worker_api {
         worker_proposal_api_object() {
         }
 
-        void fill_meta(const worker_proposal_object& wpo) {
+        void fill_main_fields(const worker_proposal_object& wpo) {
```

Comment:
In name should be some information about `worker` - it should be useful for the future developers.